app/views/hours/_form.html.erb

```diff
@@ -0,0 +1,24 @@
+<div class="row">
+  <div class="col-sm-6">
+    <div class="content-container">
+      <%= form_for @hour do |f| %>
+          <div class="form-group">
+            <%= f.label :title %>
+            <%= f.text_field :title, class: 'form-control' %>
+          </div>
+          <div class="form-group">
+            <%= f.label :body %>
+            <%= f.text_area :body, rows: 6, class: 'form-control' %>
+          </div>
+          <div class="form-group">
+            <%= f.submit "Save" %>
+          </div>
+      <% end %>
+    </div>
+  </div>
+  <div class="col-sm-6">
+    <div class="content-container">
+      <p>Please keep to the current format for displaying store hours. It will format the text for you.</p>
```

Comment:
same comment here about forcing a formatting and not expecting your use to. user-proof all user input